.github/workflows/release-bumper.yml

```
@@ -26,7 +26,7 @@ jobs:
       - id: set_branches
         run: |
           matrix=$(git for-each-ref --format='%(refname:short)' refs/remotes/origin | grep -e "origin/main" -e "origin/release-1.[0-9]\+$" | grep -v "release-1\.[0,1,2,3]$" | sed -r 's/origin\/(.*)/{"branch": "\1"}/g' | jq -s)
-          echo "name=matrix::{\"branches\":$(echo $matrix)}" >> ${GITHUB_OUTPUT}
+          echo "{matrix}={\"branches\":$(echo $matrix)}" >> $GITHUB_OUTPUT
```

I think we should drop the curly braces from both sides; e.g. 
```suggestion
          echo "matrix=\"branches\":$(echo $matrix)" >> $GITHUB_OUTPUT
```

---

i think the curly brackets on the RHS should stay, because the value is a json, that gets read in line 37 below.